go/pkg/bertyaccount/service_account.go

```diff
@@ -171,10 +172,13 @@ func (s *service) openAccount(req *OpenAccount_Request, prog *progress.Progress)
 }
 
 // OpenAccount starts a Berty node.
-func (s *service) OpenAccount(_ context.Context, req *OpenAccount_Request) (*OpenAccount_Reply, error) {
+func (s *service) OpenAccount(ctx context.Context, req *OpenAccount_Request) (_ *OpenAccount_Reply, err error) {
 	s.muService.Lock()
 	defer s.muService.Unlock()
 
+	_, _, endSection := tyber.Section(ctx, s.logger, fmt.Sprintf("Opening account %s (AccountService)", req.AccountID))
+	defer func() { endSection(err, "") }()
```

Comment:
Can you double check that if you put an `err = fmt.Errorf()` below, it will be ported here too, I've a doubt that you need to use a `var err *error` manually

Reply:
`err` is never set since `err := s.openAccount(req, nil)` use `:=` instead of `=` bellow

Reply:
see this playground https://play.golang.org/p/ejXulLs39tt